DQMOffline/Trigger/python/WprimeMonitor_cfi.py

```diff
@@ -0,0 +1,18 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.topMonitoring_cfi import topMonitoring
+
+hltWprimemonitoring = topMonitoring.clone()
+hltWprimemonitoring.FolderName = cms.string('HLT/TopHLTOffline/TopMonitor/default/')
+
+hltWprimemonitoring.histoPSet.ptPSet = cms.PSet(
+  nbins = cms.uint32(  45   ),
+  xmin  = cms.double(   100   ),
+  xmax  = cms.double(  1000  ),
+)
+
+
+hltWprimemonitoring.histoPSet.elePtBinning = cms.vdouble(100.,110.,120.,130.,140.,150.,160.,170.,180.,190.,200.,220.,240.,260.,280.,300.,350.,400.,450.,1000.)
```

Comment:
@mtosi I understand that we are not doing the SF for the analysis. Everyone has the different perception regarding the binning, so for the events which will be coming between 0 and 100 GeV would be noise and thats not related to this path, so I thought for this trigger we are interested in the behaviour of this path, which we would see at high electron pt.